src/lib/utils.js

```diff
@@ -0,0 +1,71 @@
+import rc from 'rc';
+import fs from 'fs';
+import gitRemoteOriginUrl from 'git-remote-origin-url';
+import getUrlFromGitRemote from './git';
+import loadPropertiesFile from './properties-file';
+
+const gitlabEnvFileName = 'gitlab.env.yml';
+
+async function getConf(cmd, bSingle = false) {
+  const conf = rc('gitlab');
+  const errors = [];
+
+  // Check for token
+  if (!conf.token) {
+    if (!process.env.GITLAB_TOKEN) {
```

Comment:
We shouldn't need this, the [rc](https://www.npmjs.com/package/rc) package will automatically look for environment variables:

> environment variables prefixed with ${appname}_

I feel like that'd mean `GITLAB_TOKEN` would already register... unless it's case sensitive and looking for `gitlab_token`. Did you have problems without this code?

Reply:
It is case sensitive, e.g. if env variable `GITLAB_TOKEN` is set, it would be exposed as `conf.TOKEN`

Reply:
I didn't know rc could do that. However, doing so will lead to not knowing from where comes the token, command arguments or environment variable. Don't you think this is a problem ?

Reply:
I don't think it's a problem, but I'm happy to support it.

Reply:
Agreed with Baptiste. Doing so saves some lines code, but takes more brain power to understand for the next contributor.